src/lib/tabwidget/tabwidget.cpp

```diff
@@ -787,8 +787,17 @@ QByteArray TabWidget::saveState()
         if (!webTab)
             continue;
 
-        WebTab::SavedTab tab(webTab);
-        tabList.append(tab);
+        // save state does not support saving restore tab
```

Comment:
@nowrep So I hope you will fix it :D , maybe we have to skip `view-source` tabs, too. As an example QupZilla crashes when switching to a restored view-source tab of a youtube video.